library/src/events.rs

```diff
@@ -69,11 +69,19 @@ pub struct PatchEvent {
 
     /// When this event occurred as a Unix epoch timestamp in seconds.
     pub timestamp: u64,
+
+    /// An optional message to be sent with the event.
+    pub message: Option<String>,
```

Comment:
Leave yourself a comment about the possible contents here, and that it's never PII.

Reply:
Done